ui/framework/src/appui-react/accudraw/AccuDrawCommandItems.ts

```diff
@@ -28,7 +28,7 @@ import { ToolItemDef } from "../shared/ToolItemDef";
 import { BumpToolSetting, FocusToolSettings } from "../tools/ToolSettingsTools";
 
 /** AccuDraw Command Items - useful in Keyboard Shortcuts
- * @beta
+ * @public
```

Comment:
AccuDraw UI was recently promoted from @alpha to @beta. It is not ready for @public yet.